src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -188,435 +188,730 @@ If a problem occurs, check output in /var/log/proactive/scheduler
-MYSQL_ROOT_PASSWORD=change_me
-DB_RM_PASS=change_me
-DB_SCHEDULER_PASS=change_me
-DB_CATALOG_PASS=change_me
-DB_PCA_PASS=change_me
-DB_NOTIFICATION_PASS=change_me
-
-----
-
-
-==== Define and configure ProActive containers
-Once you create and configure the `.env` file, you can select one of the three following Docker Compose configurations to start the ProActive server and Node containers with the desired database.
-
-We currently provide support for three Database systems, namely HSQLDB, PostgreSQL, and MySQL.
-
-
-
-===== Set up ProActive using embedded HSQL Database
-This Docker Compose configuration `docker-compose-hsqldb.yml` can be used to run the ProActive server container with an embedded HSQL Database.
-It will start up to two containers, one for the server and one for the Nodes.
 
-[source]
-----
-version: '2.1'
-
-services:
-  proactive-server:
-    restart: on-failure
-    image: dockerhub.activeeon.com/${RELEASE}/proactive-scheduler:${PROACTIVE_VERSION}
-    extra_hosts:
-      - service-node:${HOST_ADDRESS}
-    container_name: proactive-server
-    networks:
-      - pa-network
-    ports:
-      - ${PORT}:${PORT}
-      - ${PAMR_PORT}:${PAMR_PORT}
-    env_file:
-      - .env
-    environment:
-      - PROTOCOL
-      - PORT
-    volumes:
-      - proactive-default:/opt/proactive/server/default
-      - proactive-previous:/opt/proactive/server/previous
-    healthcheck:
-        test: ["CMD", "curl", "-k", "-f", "${PROTOCOL}://proactive-server:${PORT}"]
-        interval: 30s
-        timeout: 10s
-        retries: 30
-
-  proactive-node:
-    image: dockerhub.activeeon.com/${RELEASE}/proactive-node:${PROACTIVE_VERSION}
-    hostname: service-node
-    container_name: proactive-node
-    networks:
-      - pa-network
-    env_file:
-      - .env
-    # enable DOCKER_HOST on Windows machine when you want to run docker-in-docker
-    #environment:
-    #    - DOCKER_HOST=tcp://host.docker.internal:2375
-    volumes:
-      - /tmp
-      - /var/run/docker.sock:/var/run/docker.sock:rw
-      - node-default:/opt/proactive/node/default
-      - node-previous:/opt/proactive/node/previous
-    depends_on:
-      proactive-server:
-        condition: service_healthy
 
-volumes:
-  proactive-default:
-  proactive-previous:
-  node-default:
-  node-previous:
+==== Configure the Docker Compose Input File
+
+Before starting the ProActive installation, you need first to configure the installation input parameters using the Docker Compose environment file `.env`.
 
-networks:
-  pa-network:
+'''
+.*[underline blue]#Click here to display the .env input file#*
+[example%collapsible]
+====
+link:./references/automatedInstallScripts/.env[.env, title="Click to download"^]:
+[source,yaml]
 ----
+include::./references/automatedInstallScripts/.env[]
+----
+====
+'''
 
+===== Required configuration
+The `.env` file referenced above can be used as it is for a minimal installation. However, it requires the definition of five mandatory parameter values:
 
```

Comment:
The `.env` file referenced above can be used as is for a minimal installation.

Reply:
I used this sentence:

The `.env` file mentioned above is ready for use in a minimal installation. However, it does require the definition of five mandatory parameter values: